examples/write_parquet_cloud/Cargo.toml

```diff
@@ -7,4 +7,6 @@ edition = "2021"
 
 [dependencies]
 aws-creds = "0.36.0"
-polars = { path = "../../crates/polars", features = ["lazy", "aws", "parquet", "cloud_write"] }
+polars = { path = "../../crates/polars", features = ["lazy", "aws", "parquet", "cloud_write", "streaming"] }
```

Comment:
Why does this need `streaming`?

Reply:
Because I'm getting this error when `streaming` is disabled:

```
thread 'main' panicked at /Users/philip/polars/crates/polars-lazy/src/frame/mod.rs:625:17:
activate feature 'streaming'
```

Reply:
Ah.. Fair enough!